synfig-studio/plugins/lottie-exporter/lottie-exporter.py

```diff
@@ -8,9 +8,12 @@
 
 Supported Layers are mentioned below
 """
+
 import os
-import json
 import sys
+sys.path.append(os.abspath(os.dirname(__file__)))
```

Comment:
Looks like this should be `os.path.abspath()`

Reply:
Ouch!